deno/get_price/src/mod.ts

```diff
@@ -0,0 +1,77 @@
+export default async function (req: any, res: any) {
+  const CoinAPIkey = req.env["COIN_API_KEY"];
+  const AlphavantageAPIkey = req.env["ALPHAVANTAGE_API_KEY"];
+  const GOLDAPIkey = req.env["GOLD_API_KEY"];
+
+  const { type } = req.payload;
+
+  if (!type) {
+    throw new Error("type is required");
```

Comment:
Please make sure to return a response like documented in the issue rather than throwing an error.